specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/examples/DscpConfigurationCreate.json

```diff
@@ -0,0 +1,156 @@
+{
+    "parameters": {
+      "api-version": "2020-06-01",
+      "subscriptionId": "subid",
+      "resourceGroupName": "rg1",
+      "dscpConfigurationName": "mydscpconfig",
+      "parameters": {
+        "properties": {
+          "qosTrafficProfiler": {
+            "properties": {
+              "markings": [46],
+              "sourceIpRanges": [
```

Comment:
Since all of these properties are list object, could we use more than one item in the examples?

Reply:
Updated accordingly

Reply:
Not sure why it is not showing as outdated.